src/store/ChartStore.js

```diff
@@ -146,6 +146,7 @@ class ChartStore {
     startUI() {
         const stxx = this.stxx;
         stxx.chart.allowScrollPast = false;
+        stxx.chart.allowScrollFuture = false;
```

Comment:
@brucebinary  I think we can also take this from chart props. from the library user.

Reply:
should the need arises yes, but I think it's better for the user to set the range instead (I just find out about this today): http://documentation.chartiq.com/tutorial-Understanding%20Chart%20Range.html#main

I can't think of a use case where users would just want to scroll indefinitely to emptiness